app/scripts/views/mixins/password-mixin.js

```diff
@@ -10,7 +10,8 @@ define(function (require, exports, module) {
 
   module.exports = {
     events: {
-      'change .show-password': 'onPasswordVisibilityChange'
+      'change .show-password': 'onPasswordVisibilityChange',
+      'keyup input.password': 'onPasswordKeypress'
```

Comment:
Since the event is `keyup`, can you rename the function to `onPasswordKeyUp`? That'll help tracing the source of the event a little easier.